app/src/main/java/tech/vee/veecoldwallet/Activity/ImportSeedActivity.java

```diff
@@ -115,10 +116,17 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                 case 0:
                     Toast.makeText(activity, "Cancelled", Toast.LENGTH_LONG).show();
                     break;
-
                 case 2:
-                    String seed = QRCodeUtil.parseSeed(qrContents);
-                    if (VEEWallet.validateSeedPhrase(activity, seed)) {
```

Comment:
validateSeedPhrase() must be called to distinguish between normal seed and foreign seed. Here you have removed this logic.